djangocms_versioning/cms_config.py

```diff
@@ -129,7 +130,21 @@ def handle_content_model_manager(self, cms_config):
         for versionable in cms_config.versioning:
             replace_default_manager(versionable.content_model)
 
+    def handle_admin_field_modifiers(self, cms_config):
+        """Allows for the transformation of a given field in the ExtendedVersionAdminMixin
+        """
+        extended_admin_field_modifiers = getattr(cms_config, "extended_admin_field_modifiers", None)
+
+        if not isinstance(extended_admin_field_modifiers, list):
+            raise ImproperlyConfigured("extended_admin_field_modifiers must be list of tuples")
+
+        for model, field, method in extended_admin_field_modifiers:
+            self.add_to_field_extension[model] = dict()
```

Comment:
Supplying a dict as mentioned removes the need to do this because the extended_admin_field_modifiers will already be in this format.